pkg/controller/machine/controller.go

```diff
@@ -167,6 +173,41 @@ func (r *ReconcileMachine) Reconcile(ctx context.Context, request reconcile.Requ
 	// This must be a copy otherwise the referenced slice will be modified by later machine conditions changes.
 	originalConditions := conditions.DeepCopyConditions(m.Status.Conditions)
 
+	// Check Status.AuthoritativeAPI
+	// If not MachineAPI. Set the paused condition true and return early.
+
+	// @joelspeed i think we want some defaulting here, i.e if unset then default
+	// to MAPI, as it's not currently set. not sure if that's part of another
+	// card for setting the status / looking at the migration logic or best done
+	// here. Currently, if the field is unset we will set the paused condition
+	// and not reconcile.
+
+	if m.Status.AuthoritativeAPI != machinev1.MachineAuthorityMachineAPI {
```

Comment:
I'm not sure if this is the exact behaviour we want. Currently if the `AuthoritativeAPI` is unset we will pause and not reconcile. 

I figure 'i'm not sure so do nothing' is an OK approach, but let me know if the plan is different :)

Reply:
The card says:
```
* Should not reconcile when .status.authoritativeAPI is not MachineAPI
* Except when it is empty (prior to defaulting migration webhook)
```
Also, note, the feature work here should be gated on the presence of the feature gate being enabled, we need to work that in here as well

Reply:
Ah 🤦  - ok I'll update the code/tests !

Reply:
Don't think this one has been updated?

Reply:
Yep not done this yet - going to feature gate + this once tests are done :D